doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3677,6 +3677,12 @@ Trusted Firmware-M (TF-M)
 
 The issues in this section are related to the TF-M implementation in the |NCS|.
 
+.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0
+
+NCSDK-22907: TF-M does not build with :kconfig:option:`CONFIG_TFM_ITS_ENCRYPTED` enabled and :kconfig:option:`CONFIG_TFM_PARTITION_PROTECTED_STORAGE` disabled.
+
```

Comment:
Do we have more info here? Any error messages that come and what is causing this?

Reply:
Added and the final part of the error message